dashboard/app/controllers/foorm/simple_survey_controller.rb

```diff
@@ -69,7 +69,7 @@ def show
           formName: form_data[:form_name],
           formVersion: latest_version,
           surveyData: params[:survey_data] || form_data[:survey_data],
-          submitApi: MISC_FOORM_SUBMIT_API,
+          submitApi: FOORM_SIMPLE_SURVEY_SUBMIT_API,
```

Comment:
Since we're renaming things, do we want to have foorm in the names of things? since the simple survey api is intended to be form technology agnostic

Reply:
I don't know if it's form technology agnostic if it's stored within the Foorm module :). I am definitely up to discuss this question generally though, I'm trying to come up with some opinions about how to describe things. This work has added the term "Survey" to my Foorm lexicon, which is something like:

Survey: a data collection exercise -- uses a Form to collect information, and generates Submissions
Foorm: Code.org's system for administering Surveys.
Form, Submission: entities used in by our Foorm system.

Reply:
that makes sense!